src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -344,13 +344,17 @@ class SidebarPanels extends React.Component<Props, State> {
                 {hasMetadata && (
                     <Route
                         exact
-                        path={`/${SIDEBAR_VIEW_METADATA}`}
-                        render={() => {
+                        path={[
+                            `/${SIDEBAR_VIEW_METADATA}`,
+                            `/${SIDEBAR_VIEW_METADATA}/filteredTemplates/:filteredTemplateIds?`
+                        ]}
+                        render={({match}) => {
                             this.handlePanelRender(SIDEBAR_VIEW_METADATA);
                             return isMetadataSidebarRedesignEnabled ? (
                                 <LoadableMetadataSidebarRedesigned
                                     elementId={elementId}
                                     fileId={fileId}
+                                    filteredTemplateIds={match.params.filteredTemplateIds}
```

Comment:
From a seperation of concerns standpoint it's probably best to pass an array of template ids here instead of a comma delimited string. You can do the string parsing here and just pass the resulting array as a prop.